_posts/2016-08-01-tips-for-freshmen-by-nuswhispers.markdown

```diff
@@ -0,0 +1,144 @@
+9. Sometimes, the friends you make in school lurk unassumingly in lecture halls, CCAs, gym, random smiles at the bus-stop, common modules taken, friend intro etc etc. I made a very good friend for one of the modules taken during Y1S1. While he was an introvert (or so i thought), I made small talk with him and before I knew it, we became fast friends.
+The number of good friends I've made in NUS can be counted on one hand. But they have provided me with valuable information, laughter, HTHT sessions and whatnot. These friendships were all forged in the above mentioned places and sadly not in camps ha ha. In life, you only need a few good friends that have your back when you need it. That and your Significant Other HA HA.  
+1. I believe I triggered the OCD in some of you when you noticed bullet point one was missing at the beginning of this post.
+That is good. Attention to detail is very important. Ok i digress again.
+>
+>Anyway you can skip the previous points mentioned because this point is the most important.
+And that is to enjoy what you are learning in school. If you don't like your major, change it don't drag. If you find some modules you've taken to be boring, make it interesting (or drop it before week 5?). Don't dwell on the past (failures? bad grades?) or worry too much about the future. Enjoy learning in the present. It is a gift. Time lost which cannot be recovered.
+Anyway this post is freaking long so imma just peace out.. gotta live my life now :D
+>
+> Cheers,
+>
+> Your senior
+
+
+- Get the Proxy Bookmarklet installed on your laptops/desktops. The instructions for installation on both Windows/Mac devices can be found here: <http://libguides.nus.edu.sg/proxybookmarklet/installondesktop>. Once installed, if you ever come across a journal article in an academic journal that you wish to access, just go to your bookmarks and click on the proxy bookmarklet to gain free access to the article by signing in with your NUSNET ID. These articles help to complement your learning, and also help one with writing good essays/assignments. [[NUSWhispers #22833](https://www.nuswhispers.com/confession/22833)]
+
+- Get EndNote! This is a referencing software that helps you to manage and format your citations when you are writing your assignments and essays. They have a ton of reference styles uploaded within their database and you’ll never need to type out your references manually ever again. Without EndNote, I would have never finished my thesis. Instructions for installations are found here: <http://libguides.nus.edu.sg/endnote/installendnote >[[NUSWhispers #22833](https://www.nuswhispers.com/confession/22833)]
+
+- If you don't wish to splash out money on a new textbook, search the used textbook forum in IVLE or just use the RBR copy. [Han Ming Guang]
+
+- Here is how you get access of textbooks legally without spending extra money:  
+1. Go to NUS library website (google "NUS library")
```

Comment:
This one is quite bothersome because of the way the OP wrote the points (it starts from 2 to 9 then back to 1). The markdown will automatically list it in ordered form from 1 to 9, ignoring the numbers in the actual post (https://daringfireball.net/projects/markdown/syntax#list under ordered lists). I wanted to keep the cheekiness of the post and if I were to make it as a sub-bullet it will have it as ordered list :(